configs/systemd/dist-git-gc.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Git Repositories Clean Up
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/garbage_collection 2
```

Comment:
I would prefer to name this executable dist-git-gc

Reply:
Can we please move the <depth> number to the configuration file as well?